qt/scientific_interfaces/Indirect/IndirectFittingModel.cpp

```diff
@@ -688,11 +688,17 @@ IndirectFittingModel::createSequentialFit(const IFunction_sptr &function,
   fitAlgorithm->setProperty("StartX", startX.str());
   fitAlgorithm->setProperty("EndX", endX.str());
 
-  auto excludeRegion =
-      m_fitDataModel->getExcludeRegionVector(FitDomainIndex{0});
-  if (!excludeRegion.empty()) {
-    fitAlgorithm->setProperty("Exclude", excludeRegion);
+  std::stringstream excludeRegions;
+  for (size_t i = 0; i < m_fitDataModel->getNumberOfDomains(); i++) {
+    if (!m_fitDataModel->getExcludeRegionVector(FitDomainIndex{i}).empty()) {
+      excludeRegions << m_fitDataModel->getExcludeRegion(FitDomainIndex{i})
+                     << ",";
+    } else {
+      excludeRegions << "0.0,0.0,";
```

Comment:
I just want to check because I am not sure, should we avoid putting a comma at the very end of the `excludeRegions` string?

Reply:
It is a valid input to a mantid array propperty. I do the same with the startX and endX vector strings